OxfordBodleian/Juel-Jensen/BDLaethg36.xml

```diff
@@ -102,90 +95,304 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <locus from="39v" to="40r"/>
+                                ኦእግዝእትየ፡ <hi rend="rubric">ማርያም</hi>፡ ዕቀብኒ፡ ለገብርኪ፡ ገብረ፡ 
+                                <hi rend="rubric"><del rend="effaced">ጊዮርጊስ<supplied reason="omitted">፡</supplied></del></hi>
+                                ወጸሐፊሁ፡ ኃትዕ፡ ገብረ፡ <hi rend="rubric">ሚካኤል።</hi>
+                            </explicit>
+                        </msItem>
+
+                        <msItem xml:id="ms_i2">
+                            <locus from="40r" to="52v"/>
+                            <locus from="54r" to="54v"/>
+                            <title type="complete" ref="LIT2444Temher"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez" type="inscriptio">
+                                <locus target="#40r"/>
+                                <hi rend="rubric">በእንተ፡ ትምህርተ፡ <sic>ኀ</sic>ቡአት፡ 
+                                    ቅድመ፡ ዘ</hi>ትትነገር፡ እም<sic>ጰ</sic>ርስፎራ፡ ለምእመናን፡ ህቡአት<hi rend="rubric" rendition="#partialRubric"
+                                        >፨</hi>
+                            </incipit>
+                            <incipit xml:lang="gez">
+                                <locus from="40r" to="40v"/>
+                                ትምሕርተ፡ ህቡአትሰ፡ ከመ፡ ዝንግር፡ ዘቅድመ<supplied reason="omitted">፡</supplied> 
+                                ሀ<unclear>ሎ</unclear>፡ ወ<unclear>ይ</unclear>ሄሉ፡ ዘመፅአ፡ ዘ<pb n="40v"/>ሐመ፡ 
+                                ወሞተ፡ ወተቀብረ፡ ወተንሥአ፡ ፃዕረ፡ ዘሞት፡ ፈትሐ፡ ወዘእሙታን<hi rend="rubric" rendition="#partialRubric"
+                                    >፨</hi> ተንሥአ<hi rend="rubric" rendition="#partialRubric"
+                                        >፨</hi>
+                            </incipit>
+                            
+                            <explicit xml:lang="gez">
+                                <locus target="#54r"/>
+                                <hi rend="rubric">በከመ፡</hi> <sic resp="JK">ቤይ፡</sic> ሙሴ፡ <sic>ዳባዕዳንሂ፡</sic> ነቢያት፡ ንሕነሂ፡ 
+                                አኃውየ፡ ቦቱ፡ ተወኪለነ፡ ነሀብ፡ ሎቱ፡ ስብሐት፡ ወጽንዕ፡ ለዓለመ፡ ዓለም።።።
+                            </explicit>
+                            <explicit xml:lang="gez" type="supplication">
+                                <locus target="#54v"/>
+                                ዕቀብኒ፡ ወአድኅንኒ<hi rend="rubric" rendition="#partialRubric">፨</hi> እመከራ፡ ሥጋ፡ ወነፍስ፡ 
+                                ሀጥ<supplied reason="omitted" resp="JK">እ</supplied>፡ ወድኩም፡ 
+                                ለገብርከ<hi rend="rubric" rendition="#partialRubric">፨</hi> 
+                                ገብረ<hi rend="rubric" rendition="#partialRubric">፨</hi> 
+                                <hi rend="rubric"><del rend="effaced" unit="chars" quantity="3"/>ን፡ ስ።</hi>
+                                ወፀሃፊሁ<hi rend="rubric" rendition="#partialRubric">፨</hi> <hi rend="rubric">ኪዳነ፡ ማርያም፡</hi> 
```

Comment:
Given the previous suggestions, also the scribe ኪዳነ፡ ማርያም፡ should perhaps be marked up